base-template/base-template.md

```diff
@@ -1,77 +1,123 @@
+{Template author tip:
 
-# Base template
+* This base template captures the structure and common elements that all doctype templates should follow. It provides guidance for template authors to follow when creating a new template.
+* “Template author and template reviewer tips” should be removed from the final {doctype}-template.md. Other tips should remain.
+* Tweak other tips to align with the doctype you are writing for.
 
-## Overview
+}
 
-The base captures the structure and common elements that all types of documentation templates associated with The Good Docs Project (GDP) should follow. It includes guidance for downstream template authors to follow when creating a new template.
+{Document author tip:
 
-In our practice, the most commonly used documentation types are of concept, task, and reference. Any document that you create is likely to either be one of these types or a combination of a few of them. With the help of the base template, you can create a uniform, regulated template for any of these documentation types.
+* This {doctype} template captures the structure and common elements that all {doctype} documents should follow.
+* This template includes tips (such as this) and variables inside curly brackets. These tips and variables must be replaced or removed from the final document.
 
+}
 
-## Before you start
-To be able to make the best of this base template,  you should have sufficient knowledge of:
+{Document author tip:
 
-1. Technical documentation types - to be able to identify the type of documentation you need to create a template for, or modify the existing one into.</li>
-2. Writing technical documentation - to accurately and clearly express instructions to the reader.</li>
+* All templates must include snippets of [structured data](https://developers.google.com/search/docs/guides/sd-policies). to help search engines and facilitate content reuse. 
```

Comment:
```suggestion
* All templates must include snippets of [structured data](https://developers.google.com/search/docs/guides/sd-policies) to help search engines and facilitate content reuse. 
```
And is the "must" really necessary? What if the author has no need for snippets?